DevTools/ContainerDebug/NanoDockerBuild.cmd

```diff
@@ -4,12 +4,10 @@ SETLOCAL EnableDelayedExpansion
 set LOCAL_BIN_STAGING_DIR=BinariesToCopy
 set VS_REMOTE_DEBUGGER_BIN="%VSINSTALLDIR%CoreCon\Binaries\Phone Tools\Debugger\target\x64"
 set VS_REMOTE_DEBUGGER_LIB="%VSINSTALLDIR%CoreCon\Binaries\Phone Tools\Debugger\target\lib"
-set VS_CRT_REDIST_REL[0]="%VcToolsRedistDir%onecore\x64\Microsoft.VC141.CRT"
-set VS_CRT_REDIST_REL[1]="%VcToolsRedistDir%onecore\x64\Microsoft.VC150.CRT"
-set VS_CRT_REDIST_DBG[0]="%VcToolsRedistDir%onecore\debug_nonredist\x64\Microsoft.VC141.DebugCRT"
-set VS_CRT_REDIST_DBG[1]="%VcToolsRedistDir%onecore\debug_nonredist\x64\Microsoft.VC150.DebugCRT"
+set VS_CRT_REDIST_REL="%VcToolsRedistDir%onecore\x64\Microsoft.VC142.CRT"
+set VS_CRT_REDIST_DBG="%VcToolsRedistDir%onecore\debug_nonredist\x64\Microsoft.VC142.DebugCRT"
 
-set UCRT_DLL_PATH="%WindowsSdkBinPath%\x64\ucrt"
+set UCRT_DLL_PATH="%WindowsSdkVerBinPath%\x64\ucrt"
```

Comment:
Interesting, I can see ucrtbased.dll in both directories, albeit their sizes are different. Do you not have one in the former path or it was not working for you?

Reply:
The former path does not even exist on my box. Someone else commented to the same effect on a blog post about this script a few years ago:
https://blogs.technet.microsoft.com/nanoserver/2017/02/24/leverage-containers-to-test-and-debug-native-applications-on-nano-server/#comment-23536

Reply:
This is what I have in the former path, though it might be that this was installed by ADK or WDK:
```
C:\Program Files (x86)\Microsoft Visual Studio>dir "%WindowsSdkBinPath%\x64\ucrt"
 Volume in drive C is Windows
 Volume Serial Number is B6CE-3457

 Directory of C:\Program Files (x86)\Windows Kits\10\bin\x64\ucrt

03/09/2017  03:59 PM    <DIR>          .
03/09/2017  03:59 PM    <DIR>          ..
01/06/2017  10:40 AM         1,821,376 ucrtbased.dll
               1 File(s)      1,821,376 bytes
               2 Dir(s)  98,576,322,560 bytes free
```
Not sure why, but I do not seem to be able to see the comment. 

Can you also make this into a loop to make sure we do not break earlier folks?

Reply:
Switched to a loop.

It looks like they moved sites and lost the comments. Here's an archived version:
https://cc.bingj.com/cache.aspx?q=nanoserver+windowssdkbinpath+debug&d=4975895837279218&mkt=en-US&setlang=en-US&w=8qJ5oeFemJTWxrJsT_xGxhvtd2MiSdCE